lua/bufferline/icons.lua

```diff
@@ -16,25 +15,26 @@ local hl_groups = {}
 -- changes, therefore we need to re-define colors for all groups we have
 -- already highlighted.
 local function set_highlights()
-  for i, hl_group in ipairs(hl_groups) do
+  for _, hl_group in ipairs(hl_groups) do
```

Comment:
You'll see a lot of unused variables go to `_` in this PR as well— it helps quiet down the LSP and frees up the name for other use.